hermes-management/src/main/java/pl/allegro/tech/hermes/management/config/SchemaRepositoryAuthorizationProperties.java

```diff
@@ -0,0 +1,33 @@
+package pl.allegro.tech.hermes.management.config;
+
+public class SchemaRepositoryAuthorizationProperties {
```

Comment:
I'd reuse `SchemaRepositoryAuthorizationProperties` from `hermes-common` instead of adding this one.